tests/test_admin.py

```diff
@@ -92,3 +93,27 @@ def test_admin(app):
             data={'rowid': -42, 'action': 'activate'},
             follow_redirects=True
         )
+
+        # Test empty mail form
+
+        res = client.post(
+            url_for("user.create_view"),
+            data={'email': ''},
+            follow_redirects=True
+        )
+        assert b'This field is required.' in res.data
+
+        # Test user creation
+
+        res = client.post(
+            url_for("user.create_view"),
+            data={'email': 'test2@test.cern'},
+            follow_redirects=True
+        )
+        assert _datastore.get_user('test2@test.cern') is not None
+
+        user_data = dict(email="test3@test.cern", active=False,
+                         password=encrypt_password('aafaf4as5fa'))
+        _datastore.create_user(**user_data)
```

Comment:
IMHO this should be part of new test case that simulates flow defined in https://github.com/inveniosoftware/invenio-accounts/issues/154